### PR TITLE
Feature/swagger

### PR DESCRIPTION
**Pull request by steven-lotero**
## Summary
- Added Swageer dependencies
- Added Swagger to the FindOne method of the UserController
- added Swagger to the birthday object of the user model
## Trello Card
https://trello.com/c/Wl4UuNJY/40-swagger

### CODE REVIEW

**Review by CarlosLugo22 — APPROVED** [review]
Very well

File: src/main/java/wolox/training/controllers/UserController.java [hunk 1]
```diff
@@ -1,5 +1,10 @@
 package wolox.training.controllers;
 
+import io.swagger.annotations.Api;
```
Comment (JaimLus) [inline src/main/java/wolox/training/controllers/UserController.java:3]: Import sin usar mi hermano.

File: src/main/java/wolox/training/controllers/BookController.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package wolox.training.controllers;
 
+import io.swagger.annotations.Api;
```
Comment (JaimLus) [inline src/main/java/wolox/training/controllers/BookController.java:3]: Import sin usar 😄

**Review by JaimLus — APPROVED** [review]
Excelente trabajo!

File: src/main/java/wolox/training/controllers/UserController.java (hunk 1 above)
Comment (steven-lotero) [reply src/main/java/wolox/training/controllers/UserController.java:3]: Mala mía, será corregida junto con los cambios solicitados

**Pull request merged**